### PR TITLE
Add calculator tool

### PR DESCRIPTION
**Pull request by nfcampos**
With a calculator tool the agent test now passes

### CODE REVIEW

File: langchain/package.json [hunk 1]
```diff
@@ -78,6 +78,7 @@
   },
   "dependencies": {
     "exponential-backoff": "^3.1.0",
+    "expr-eval": "^2.0.2",
```
[the remark this answers is not in the document]
Comment (hwchase17) [reply langchain/package.json:81]: eh how heavy of a dependency is it? if its light its probably fine to include

[the remark this answers is not in the document]
Comment (nfcampos) [reply langchain/package.json:81]: It's self contained, no dependencies of its own, so probably fine to leave as is? https://github.com/silentmatt/expr-eval

File: langchain/agents/tests/agent.test.ts [hunk 2]
```diff
@@ -3,10 +3,11 @@ import { OpenAI } from "../../llms/openai";
 import { loadAgent } from "../load";
 import { AgentExecutor, Tool } from "../index";
 import { SerpAPI } from "../tools/serpapi";
+import { Calculator } from "../tools/calculator";
 
 test("Run agent from hub", async () => {
   const model = new OpenAI({});
-  const tools: Tool[] = [SerpAPI()];
+  const tools: Tool[] = [SerpAPI(), Calculator];
```
Comment (nfcampos) [inline langchain/agents/tests/agent.test.ts:10]: Wasn't sure about this one? Tool interface seems to be an object, but the SerpAPI is implemented as a function that returns that object. Do we want them all to be "functions returning objects"? In which case maybe Tool should just be a regular class?

File: langchain/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (hwchase17) [reply langchain/package.json:81]: yup sg

[the remark this answers is not in the document]
Comment (nfcampos) [reply langchain/package.json:81]: Btw this is quite a lot safer than the one in the python library, might make sense to revisit the ones in python at some point, arbitrary evaluation of code sounds scary, especially in production settings

File: langchain/agents/tests/agent.test.ts (hunk 2 above)
Comment (nfcampos) [reply langchain/agents/tests/agent.test.ts:10]: I'll do that in a separate PR, and then we can decide

Comment (hwchase17) [reply langchain/agents/tests/agent.test.ts:10]: sounds awesome

Comment (nfcampos) [reply langchain/agents/tests/agent.test.ts:10]: https://github.com/hwchase17/langchainjs/pull/9

**Pull request merged**